src/test/cypress/integration/exam/ExamAssessment.spec.ts

```diff
@@ -0,0 +1,186 @@
+import { generateUUID } from '../../support/utils';
+import { artemis } from '../../support/ArtemisTesting';
+import { CypressExamBuilder } from '../../support/requests/CourseManagementRequests';
+import { GROUP_SYNCHRONIZATION } from '../../support/constants';
+import dayjs from 'dayjs';
+import { ProgrammingExerciseSubmission } from '../../support/pageobjects/OnlineEditorPage';
+import partiallySuccessful from '../../fixtures/programming_exercise_submissions/partially_successful/submission.json';
+import textSubmission from '../../fixtures/text_exercise_submission/text_exercise_submission.json';
+
+// requests
+const courseManagementRequests = artemis.requests.courseManagement;
+
+// page objects
+const examStartEnd = artemis.pageobjects.examStartEnd;
+const modelingEditor = artemis.pageobjects.modelingEditor;
+const modelingAssessment = artemis.pageobjects.modelingExerciseAssessmentEditor;
+const editorPage = artemis.pageobjects.onlineEditor;
+const assessmentDashboard = artemis.pageobjects.assessmentDashboard;
+
+// Common primitives
+let uid = generateUUID();
+const courseName = 'Cypress course' + uid;
+const courseShortName = 'cypress' + uid;
+const admin = artemis.users.getAdmin();
+const student = artemis.users.getStudentOne();
+const tutor = artemis.users.getTutor();
+const packageName = 'de.test';
+let exam: any;
+let exerciseGroup: any;
+let course: any;
+let examTitle: string;
+
+describe('Exam Assessment', () => {
+    before('Create a course', () => {
+        cy.login(admin);
+        courseManagementRequests.createCourse(courseName, courseShortName).then((response) => {
+            course = response.body;
+            courseManagementRequests.addStudentToCourse(course.id, artemis.users.getStudentOne().username);
+            courseManagementRequests.addTutorToCourse(course, artemis.users.getTutor());
+        });
+    });
+
+    beforeEach('Generate new exam name', () => {
+        examTitle = 'exam' + generateUUID();
+        cy.login(admin);
+    });
+
+    afterEach('Delete Exam', () => {
+        cy.login(admin);
+        courseManagementRequests.deleteExam(exam);
+    });
+
+    after('Delete Course', () => {
+        cy.login(admin);
+        courseManagementRequests.deleteCourse(course.id);
+    });
+
+    describe('Exam exercise Assessment', () => {
+        beforeEach('Create Exam', () => {
+            prepareExam(dayjs().add(30, 'seconds'));
+        });
+
+        it('Assess a modeling exercise submission', () => {
+            courseManagementRequests.createModelingExercise({ exerciseGroup });
```

Comment:
Below you check if the title is shown in the UI, so you should set a title explicitly and use the constant value in the assertion below
```suggestion
            const exerciseTitle = 'Cypress modeling exercise';
            courseManagementRequests.createModelingExercise({ exerciseGroup }, exerciseTitle);
```

Reply:
Doesnt this go directly against the whole default value thing we are trying to set up? I would be more inclined to make the default values accessible through the response.